### PR TITLE
Added States to Overview dashboard

### PR DESCRIPTION
**Pull request by DrMichael**
Not sure, whether all went well...

### CODE REVIEW

File: grafana/dashboards/overview.json [hunk 1]
```diff
@@ -1385,6 +1591,6 @@
   },
   "timezone": "",
   "title": "Overview",
-  "uid": "kOuP_Fggz",
-  "version": 1
-}
+  "uid": "MyllnnXZk",
```
Comment (adriankumpf) [inline grafana/dashboards/overview.json]: There's no need to change the `uid`.

File: grafana/dashboards/overview.json [hunk 2]
```diff
@@ -1329,8 +1535,8 @@
       {
         "allValue": null,
         "current": {
-          "text": "http://localhost:4000",
-          "value": "http://localhost:4000"
+          "text": "https://teslamate.schmidt-schmitten.com",
```
Comment (adriankumpf) [inline grafana/dashboards/overview.json]: Please revert this

Comment (adriankumpf) [inline grafana/dashboards/overview.json]:
```suggestion
          "text": "http://localhost:4000", 
```

File: grafana/dashboards/overview.json (hunk 1 above)
Comment (adriankumpf) [reply grafana/dashboards/overview.json]:
```suggestion
  "uid": "kOuP_Fggz", 
```

**Comment by DrMichael** [comment]
> Please revert to use the default values, not the ones from your setup. In general it would be nice if only the necessary lines were changed :)

I agree. Not sure, where all those changes came from... Still there appears my URL to be there. How can I change that?!?

(Perhaps I should restart from scratch...)

File: grafana/dashboards/overview.json (hunk 2 above)
Comment (DrMichael) [reply grafana/dashboards/overview.json]: Not sure, whether the https://teslamate.schmidt-schmitten.comis completely out now...

**Comment by DrMichael** [comment]
@adriankumpf I redid it from scratch. Unfortunately I cannot keep the diff at a minimum because grafana rearranges the panels...
Perhaps zu could review and, if ok, deploy this?
[Overview-1584459056104.zip](https://github.com/adriankumpf/teslamate/files/4344270/Overview-1584459056104.zip)

**Comment by DrMichael** [comment]
I created a new pull request. Hopefully cleaner and more correct... :-)

**Pull request closed without merging**